android/hybridcommon/src/main/java/com/revenuecat/purchases/hybridcommon/mappers/PurchasesError.kt

```diff
@@ -0,0 +1,19 @@
+package com.revenuecat.purchases.hybridcommon.mappers
+
+import com.revenuecat.purchases.PurchasesError
+import com.revenuecat.purchases.hybridcommon.ErrorContainer
+
+fun PurchasesError.map(
+    extra: Map<String, Any?> = mapOf(),
+): ErrorContainer =
+    ErrorContainer(
+        code.code,
+        message,
+        mapOf(
+            "code" to code.code,
+            "message" to message,
+            "readableErrorCode" to code.name,
+            "readable_error_code" to code.name,
+            "underlyingErrorMessage" to (underlyingErrorMessage ?: ""),
```

Comment:
yes... I wanted to double check with iOS, because I think there's a userInfo is in the typescript interface but not in here. We need to double check the errors in all the hybrids I think... but that's another project.

In this case I actually just moved it to its own file and made it public (it was internal before)